ubiquity/pay.ubq.fi
# provider proxy

Resolves #235 

Wrapped `app.provider` in a `Proxy` to catch any failed call across the app

QA:

You see two page loads that require the proxy to select a new provider and retry, the third requires no retry.

You see two sets of logged failures because the app makes a call on page load, this cannot be effectively removed as a successful RPC will not need changed as seen on the 3rd page load. The proxy uses the first stored RPC and then loops the latencies retrying until successful.

https://github.com/ubiquity/pay.ubq.fi/assets/106303466/473f9885-9d8d-4bd0-8c9a-8efd5796e1db
